ref #1: fork without git history gatewatcher internal code (#2)
Gatewatcher/pytest-vulture: DEVELOPMENT.rst

pytest vulture
-------------

How it works
============

At the start of pytest execution (pytest_sessionstart),
it parses the configuration file if it exists (otherwise it takes the default values),
and then run vulture.

When a file is manages by pytest (pytest_collect_file), it looks for vulture outputs related to this file.

> If vulture outputs are found, it checks if it is an entry point and if the output is ignore by the configuration file,

Reviewer: ```suggestion
Vulture outputs are ignored if they are an entrypoint or if they are ignored by the vulture configuration file.
```

and if it is not an entry point and not ignored add the vulture output to pytest.

> If vulture no outputs are found for this file, it passes.

Reviewer: ```suggestion
If no vulture outputs are found for this file, it is skipped.
```
